staff/victor-abadia/horroy/scripts/horroy.js

```diff
@@ -0,0 +1,87 @@
+function Horroy() {
+    this.length = arguments.length;
+
+    if (arguments.length)
+        for (var i = 0; i < arguments.length; i++)
+            // this.push(arguments[i]); // WARN should avoid (if possible) calling member methods in a constructor (push)
+            this[i] = arguments[i];
+}
+
+Horroy.prototype.push = function (value) {
+    if (!value) throw Error('undefined element to push')
+
+    this[this.length++] = value;
+};
+
+Horroy.prototype.forEach = function (callback) {
+    if (typeof callback !== 'function') throw new TypeError('callback is not a function');
```

Comment:
The use of new it's not compulsory, anyways it's right, so I only comment it for you to knowing it 🤓

Reply:
in the error message I would send callback + 'is not a function', for the user to know what is not a function